hive-metastore/src/main/java/org/apache/iceberg/hive/HiveSchemaConverter.java

```diff
@@ -110,7 +111,7 @@ Type convertType(TypeInfo typeInfo) {
       case STRUCT:
         StructTypeInfo structTypeInfo = (StructTypeInfo) typeInfo;
         List<Types.NestedField> fields =
-            convertInternal(structTypeInfo.getAllStructFieldNames(), structTypeInfo.getAllStructFieldTypeInfos());
+            convertInternal(structTypeInfo.getAllStructFieldNames(), structTypeInfo.getAllStructFieldTypeInfos(), null);
```

Comment:
Should the comment parameter be an empty list instead of null? This may remove various null checks in the code e.g in `convertInternal` above
Also, for an empty struct I believe `fieldnames` and `typeinfos` will be empty instead of null so it maybe more consistent for it to be empty.

Reply:
Thanks for the review. Good point, I changed the parameter to an empty list.